lib/assets/js/jquery-front-scripts.js

```diff
@@ -94,12 +94,16 @@ jQuery( document ).ready(
 			/* Sort properties by price */
 			sort_listings : function() {
 
-				jQuery( '#epl-sort-listings' ).on(
+				jQuery( '.epl-sort-listings' ).on(
```

Comment:
Not sure what this is actually doing. Can you add docblock comments to JS? they will be minified and stripped anyway

/**
 * What this does
 *
 * @since 3.5 New thing that has changed
 */